docs/UserGuide.md

```diff
@@ -145,22 +145,30 @@ Help is also shown if you enter an incorrect command e.g. `abcd`
 
 Adds a floating task to TaskTracker.
 
-    add "FLOATING_TASK_NAME" [PRIORITY]
+    task "FLOATING_TASK_NAME" [PRIORITY]
```

Comment:
Okay, with one technical problem: the java.time.[DateTimeFormatter](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html) that I'm using does not support times like `400pm`. It needs to be like `4:00pm` instead. I'll send a PR to fix this in the User Guide.